src/Microsoft.DotNet.Watcher.Tools/Program.cs

```diff
@@ -49,47 +46,20 @@ public static int Main(string[] args)
         // To pass "--help" to the app being watched, you must use "--": dotnet watch -- --help
         internal static void SeparateWatchArguments(string[] args, out string[] watchArgs, out string[] appArgs)
```

Comment:
Do you really need to separate anything? Instead of calling this method can you just do this:

```
if (args[0] == 'help')
{
   PrintHelp();
   return;
}
// carry on using args
```
and remove this method completely?